app/src/main/java/org/p2p/wallet/history/ui/historylist/HistoryListViewPresenter.kt

```diff
@@ -28,14 +32,30 @@ class HistoryListViewPresenter(
         loadHistory()
     }
 
-    override fun loadNextHistoryPage() = Unit
+    override fun loadNextHistoryPage() {
+        launch {
+            try {
+                delay(300L)
```

Comment:
do we need this? and if yes why?

Reply:
Ohh, forgot to add message, I will remove this after you fix bug with double call method from adapter